plugins/woocommerce-admin/client/marketplace/components/product-list-content/product-list-content.tsx

```diff
@@ -1,19 +1,50 @@
-/**
- * External dependencies
- */
-
 /**
  * Internal dependencies
  */
+import ProductCard from '../product-card/product-card';
 import './product-list-content.scss';
 
-export default function ProductListContent(): JSX.Element {
+export interface Product {
+	id?: number;
+	title: string;
+	description: string;
+	vendorName: string;
+	vendorUrl: string;
+	icon: string;
+	url: string;
+	price: string | number;
+	productType?: string;
+	averageRating?: number | null;
+	reviewsCount?: number | null;
+	currency?: string;
```

Comment:
(My understanding is that) the `?` makes currency an _optional_ prop. Is that correct?

Reply:
Yes, we are not using the currency for now. We might modify the API to include the currency symbol. So this shall be updated later.